torch/_C/_distributed_c10d.pyi

```diff
@@ -271,17 +271,29 @@ class Work:
     def unbox(obj: ScriptObject) -> Work: ...
 
 class Backend:
+    class Options:
```

Comment:
why did you have to move Options into Backend? I don't see why options is relevant to any of the other apis

Reply:
No, it's because Backend has a option. The options in NCCLPG inherit from that:

Code is here:
https://github.com/pytorch/pytorch/blob/596f568592ad312fb85f22cca1618af8d327f859/torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp#L413